<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI_input</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="GB_incident_phenomena">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>411</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Sự cố của xe</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="CB_incident_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Xe đột ngột tắt máy giữa đường</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CB_incident_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Khó đề máy khi khởi động</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CB_incident_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Thấy nhiều khói đen tỏa ra khi tăng tốc</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CB_incident_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cảm giác xe đì khi leo dốc</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CB_incident_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Xe đột ngột giảm vòng tua khi tăng tốc</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CB_incident_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Có tiếng động lạ ở động cơ</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CB_incident_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vòng tua không đáp ứng ngay khi tăng </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CB_incident_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cảm giác lái bất thường, khó kiểm soát</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CB_incident_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Động cơ rung lắc mạnh khi vận hành</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_incident_phenomena_warning">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Vui lòng chọn sự cố của xe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_customer_information">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Thông tin khách hàng</string>
    </property>
    <widget class="QLabel" name="Label_customer_information_warning">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Vui lòng nhập đầy đủ thông tin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_customer_name">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>34</y>
       <width>117</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tên khách hàng</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_VIN_code">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>73</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mã VIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_number_plate">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>113</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Biển số</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_phone_number">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>152</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SĐT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_address">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>191</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Địa chỉ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fixing_date">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>231</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ngày sửa chữa</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_customer_name">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>39</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Vui lòng nhập tên của bạn</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_VIN_code">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>78</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Nhập mã VIN</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_number_plate">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>117</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Biển số xe cần kiểm tra</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_phone_number">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>156</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Sô điện thoại liên lạc</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_address">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>195</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Nhập địa chỉ nhà của bạn</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateEdit" name="DE_fixing_date">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>234</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_incident_images">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>721</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Hình ảnh sự cố</string>
    </property>
    <widget class="QGraphicsView" name="GV_inciden_images">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>39</y>
       <width>701</width>
       <height>591</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foregroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::NoDrag</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="QL_label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>710</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_confirm">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>690</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Xác nhận</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_exit">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>690</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
   <zorder>GB_incident_phenomena</zorder>
   <zorder>GB_incident_images</zorder>
   <zorder>splitter</zorder>
   <zorder>splitter_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>splitter_3</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>splitter_4</zorder>
   <zorder>GB_customer_information</zorder>
   <zorder>QL_label_3</zorder>
   <zorder>BT_confirm</zorder>
   <zorder>BT_exit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="QA_actionOpen"/>
    <addaction name="QA_actionNew"/>
    <addaction name="QA_actionSave"/>
    <addaction name="QA_actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="QA_actionGraphics_car"/>
    <addaction name="QA_actionDate"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="QA_actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="QA_actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="QA_actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="QA_actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="QA_actionGraphics_car">
   <property name="text">
    <string>Graphics car</string>
   </property>
  </action>
  <action name="QA_actionDate">
   <property name="text">
    <string>Date </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
